Travis builds failing due to out of memory error

<!--
  Please fill out each section below. This info allows CIVIC heroes to diagnose (and fix!) your issue as quickly as possible.

  Before opening a new issue, please search existing issues: https://github.com/hackoregon/civic/issues
-->

## Description

Failing builds, examples:
https://travis-ci.org/hackoregon/civic/jobs/575219394
https://travis-ci.org/hackoregon/civic/jobs/575576034

```
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

 1: 0x8dc510 node::Abort() [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

 2: 0x8dc55c  [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

 3: 0xad9b5e v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

 4: 0xad9d94 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

 5: 0xec7bf2  [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

 6: 0xec7cf8 v8::internal::Heap::CheckIneffectiveMarkCompact(unsigned long, double) [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

 7: 0xed3dd2 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

 8: 0xed4704 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

 9: 0xed7371 v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationSpace, v8::internal::AllocationAlignment) [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

10: 0xea07f4 v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationSpace) [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

11: 0x114018e v8::internal::Runtime_AllocateInNewSpace(int, v8::internal::Object**, v8::internal::Isolate*) [/home/travis/.nvm/versions/node/v10.15.3/bin/node]

12: 0x15aee6d5be1d 

error Command failed with exit code 1.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Makefile:2: recipe for target 'travis' failed

make: *** [travis] Error 1

error Command failed with exit code 2.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

The command "yarn run travis" exited with 2.
```

### Steps to reproduce

See builds

### Expected result

Builds should not run out of memory

### Actual result

Out of memory 💣

### Environment

Travis

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,7 +16,7 @@ matrix:
   - env: SUITE=2018
   - env: SUITE=COMPONENT_LIBRARY
 script:
-- yarn run travis
+- NODE_OPTIONS=--max_old_space_size=4096 yarn run travis
 after_success:
 - make deploy
 env:

